README.md

```diff
@@ -61,6 +61,17 @@ Host github.com
 
 Note: parallel downloads is only available for repositories and not forge modules.
 
+### Parallel tests
+It is also possible to use the `parallel_tests` Gem via the `:parallel_spec` task.  Use of parallelization at this level can result in large performance benefits when a system has a number of CPUs and the Rspec examples involve many full catalog compilations, such as in a control repo that compiles catalogs for every host for example.
```

Comment:
I suggest adding a warning that it may not work in all situations, such as "...but may actually result in slowdowns with small test sets (less than 100 tests)."